### PR TITLE
Bug 1136698 - Fixing keyboard tests in v2.2 r=julienw

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=04ca6c07aac5)

File: package.json [hunk 1]
```diff
@@ -48,15 +46,22 @@
...
+    "esprima": "^2.0.0",
+    "fs-extra": "^0.16.3",
+    "is-running": "^1.0.5",
     "js-yaml": "^3.2.2",
+    "jsdom-nogyp": "^0.8.3",
     "json-templater": "^1.0.1",
+    "jszip": "^2.4.0",
+    "node-uuid": "^1.4.2",
     "yargs": "^1.3.2"
   }
```
Comment (julienw) [inline package.json]: with these changes I think you need to change gaia_node_modules.revision too

File: package.json [hunk 2]
```diff
@@ -11,26 +11,24 @@
...
     "ffi-prebuilt": "~1.2.6",
-    "gaia-raptor": "0.2.0",
+    "gaia-raptor": "0.6.0",
     "grunt": "0.4.2",
     "grunt-contrib-clean": "0.5.0",
     "grunt-jsdoc": "0.5.1",
     "istanbul": "^0.3.2",
-    "jsdom-nogyp": "^0.8.3",
-    "jshint": "2.5.10",
+    "jshint": "2.6.0",
```
Comment (julienw) [inline package.json]: looks like the new jshint is not really happy with some code, see https://treeherder.mozilla.org/logviewer.html#?job_id=546130&repo=gaia-try

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e81c725d3463)

**Pull request merged**